src/components/scenes/TransactionListScene.tsx

```diff
@@ -198,25 +198,30 @@ function TransactionListComponent(props: Props) {
   })
 
   return (
-    <SceneWrapper hasTabs>
-      {SHOW_FLIP_INPUT_TESTER ? (
-        <ExchangedFlipInputTester />
-      ) : (
-        <FlashList
-          ref={flashList}
-          data={listItems}
-          estimatedItemSize={theme.rem(4.25)}
-          getItemType={getItemType}
-          keyboardShouldPersistTaps="handled"
-          keyExtractor={keyExtractor}
-          ListEmptyComponent={emptyComponent}
-          ListHeaderComponent={topArea}
-          onEndReachedThreshold={0.5}
-          refreshControl={refreshControl}
-          renderItem={renderItem}
-          stickyHeaderIndices={stickyHeaderIndices}
-          onEndReached={handleScrollEnd}
-        />
+    <SceneWrapper hasTabs hasNotifications>
+      {gap => (
+        <>
+          {SHOW_FLIP_INPUT_TESTER ? (
+            <ExchangedFlipInputTester />
+          ) : (
+            <FlashList
+              ref={flashList}
+              contentContainerStyle={{ paddingBottom: gap.bottom }}
```

Comment:
`style={ marginBottom: -gap.bottom }`

Reply:
No this is right, it should be positive. The notificationHeight is meant as an
contentContainerStyle paddingBottom (overscroll) - using it as a style padding causes a visual cutoff